NotationV0/src/sign.ts

```diff
@@ -0,0 +1,66 @@
+import * as os from 'os';
+import * as path from 'path';
+import * as taskLib from 'azure-pipelines-task-lib/task';
+
+import { AZURE_KV_PLUGIN_VERSION_FILE, NOTATION, NOTATION_BINARY, PLUGINS } from './lib/constants';
+import { getDownloadInfo, installFromURL } from './lib/install';
+
+import { getArtifactReferences } from './lib/variables';
+import { getConfigHome } from './lib/fs';
+import { getVaultCredentials } from './lib/credentials';
+import { notationRunner } from './lib/runner';
+
+export async function sign(): Promise<void> {
+    const artifactRefs = getArtifactReferences();
+    const keyid = taskLib.getInput('keyid', true) || '';
+    const signatureFormat = taskLib.getInput('signatureFormat', false) || 'cose';
+    const allowReferrerAPI = taskLib.getBoolInput('allowReferrersAPI', false);
+    const debug = taskLib.getVariable('system.debug');
+    let env = { ...process.env };
+    if (allowReferrerAPI) {
+        env["NOTATION_EXPERIMENTAL"] = "1";
```

Comment:
Will the following steps in the pipeline also turn on this `experimental` flag? Or just the current step turned it on?

Reply:
It only affects the current step.